css/style.css

```diff
@@ -1,3 +1,183 @@
-h1 {
-  color: salmon;
+body{
+  font-family: "LL brown", Arial, sans-serif ;
```

Comment:
yep I forgot to add the CDN, which is why the fonts are not that close to the homework.